modules/users/server/controllers/users/users.password.server.controller.js

```diff
@@ -138,6 +138,10 @@ exports.reset = function(req, res, next) {
 									message: errorHandler.getErrorMessage(err)
 								});
 							} else {
+							    // Remove sensitive data before login
+							    user.password = undefined;
```

Comment:
I understand removing salt, but isn't user.password is expected to be there so that passport can verify it? also where exactly is it exposed? it's going straight from code to the passport method, it's not broadcasted back to the user...